src/main.js

```diff
@@ -158,22 +152,22 @@ function getRandomIndex(array) {
 function saveCurrentPoster() {
   if (!posterSavedAlready()) {
     savedPosters.push(currentPoster)
+    savedPostersGrid.appendChild(generateMiniPosterHTML())
   }
-  displaySavedPosters()
 }
 
-function displaySavedPosters() {
```

Comment:
Lets group the function by display and data for better readability.

Reply:
Re-factored code to implement this!